plugin/evm/vm_test.go

```diff
@@ -4090,6 +4090,39 @@ func TestParentBeaconRootBlock(t *testing.T) {
 	}
 }
 
+func TestNoBlobsAllowed(t *testing.T) {
+	require := require.New(t)
+	importAmount := uint64(1000000000)
+	issuer, vm, _, _, _ := GenesisVMWithUTXOs(t, true, genesisJSONCancun, "", "", map[ids.ShortID]uint64{
```

Comment:
Confirm that this doesn't return an error.

Reply:
the ignored return value is not an error here.